### ISSUE TITLE
Bureaucratic Intervention: making the Lawyer useful and making CentComm a name to be feared

### ISSUE THREAD

**Issue by ParagonPrime** [yogstation13/yogstation-classic#1258]
# **The Problem**
- _Lawyer is an absolutely useless job in all but the best circumstances._ This is because **very** few Security members listen to Lawyers and their judgement because "lol RP what the fuck is that", in turn making Lawyer a job reserved for antag-rolling ("free sec headset!") or for an assistant with slightly more access.
- _Security is often undermanned, leaving admins to handle IC issues- to the chagrin of many._ I mention this here because part of the goal of this suggestion is an improvement on how Security operates, and their overall quality of life, which should hopefully see a major increase in Security player count.
- _Sec currently exists as little more than an official validhunter force._ I want to expand their range of duties massively, and this is part of the deal...

---
# **The Solution(s)**

Right, this is a slightly more complex and probably more controversial issue to resolve than the Knowledge Levels/Training Implants bit. But let's try, nonetheless:

Firstly,

 _**empower the Lawyer.**_
Lawyer is, as previously pointed out, howling against the wind at best and an antagroller at worst. Very few people actually listen to a lawyer and their judgement; hopefully this will be improved by improved levels of RP, but I want to take it a step further and give them a weapon by way of game mechanics. Something that has already been suggested before, in a separate issue, but is relevant to this topic as a whole:

**Give the Lawyer a fax machine able to directly communicate with CentComm.** This machine will accept any piece of paper with writing on it, or a photograph, and be able to send it to other fax machines or CentComm (with all admins recieving an alert and the ability to view the fax'd items). This is intended to procure a rapid response from CC, either directly by fax or by announcement, adressing the Lawyer's concerns. This can be anything from gross shitcurity on the part of Sec, to intense corruption from a head, to just complaining about a particular aspect of the station's administration.

However, the Lawyer can also send a special form: an Audit Request.

The Audit Request, which is sent simply by clicking the relevant option on the fax machine menu, has a **decent (but >50%) chance of spawning a CentComm Official after ~5 minutes.** Admins will also be informed, and can either force a spawn or force disable it- or leave it up to chance.

The CentComm Official will be given a primary objective: "Determine whether Research Station 13 is operating satisfactorily and report your findings to NanoTrasen Central Command". The Official will have a special PDA cartridge able to directly contact CentComm (because bluespace magic), AKA sending the message to admins, who can respond accordingly- and provide certain objectives. Including, if so necessary, a neural upload of the Nuclear Device's code.

**However, the Lawyer has a potentially even more duty now:** _legislative enforcement._

By random chance (or admin will), a message can arrive at the Lawyer's fax machine, informing him of a new policy/law that Central Command wants introduced on the station. This law can be anything from beneficial ("the clown is not to be jailed for any nonharmful shenanigans"), to annoying ("station staff are now banned from consuming junk foods"), to overly bureaucratic ("all prisoners incarcerated into the Brig, for any crime, must have records written on paper"), to downright round-changing ("all nonhumans are to be demoted from Command and Security positions"). Naturally, **it is the Lawyer's job to alert Security and the station about this law change and ensure that it is fully enforced. The law is not written up anywhere but the paper the Lawyer recieves, so use of the photocopier will be prudent.** The laws are randomized, but an admin can also write a custom one.

---

And now, for the silly part of this idea; a concept I've had for a while now:

**BureauCops.**

BureauCops would be a silly, rare gamemode (only able to be triggered when at least two admins are online, and all admins being informed beforehand, as it requires a lot of admin involvement) akin to NukeOps, with a few key differences:

A team of five CentComm Officials is deployed to SS13. Their mission: mark every single transgression, malpractice, and sign of inefficiency they can spot. Once 50 Transgressions have been marked, the CentComm officials are given the Nuclear Device code and given one command: "The employees of Research Station 13 have been marked as redundant. Release them from their contract via nuclear fission."

Now, how are Transgressions actually collected?

The Officials are given a device, a Tagger, which can remotely scan and mark any Transgression instantly; scanning a person will mark any Transgressions on them. But what exactly is a Transgression?
-Litter on the ground (including items within 10 tiles of a table)
-Uncleaned floors
-Blood-stained clothes
-Weapon left unattended outside of Brig (any weapon that is just lying somewhere outside the brig/armory)
-Weapon possessed by person without weapons license
-Unknown Individual
-"Potentially socially disruptive attire" - someone wearing a replica syndicate space suit, foam armblade, etc.
-Anything else we can think up!

Additionally, a PhotoTagger is given to the Officials- they are able to take a photo of something and send it to CentComm (an admin). The admin is then allowed to mark whether there is something that qualifies as a Transgression within the photo, and grant the Officials the point. And, of course, the Officials can always PDA CentComm directly and inform them of certain actions- which admins can also manually mark as Transgressions.

The crew is given one mission:

To last 45 minutes without being nuked- except harming or restraining the CentComm Officials is disallowed. Every time an Official is harmed (by something other than another Official), two Transgressions are added and admins informed (and encouraged to lightly BSA the perp)

At the start of the round, the "hostile activity intercept" will be replaced by a different message:

**Your station has been randomly chosen for auditing by a team of NanoTrasen Central Command-appointed inspectors!

Prepare the station and ensure it is running in perfect order! The inspectors will not hesitate to mark any transgressions they spot while operating!

NanoTrasen reminds crew that _attempting to harm or restrain the Officials in any way will be grounds for immediate termination by directed bluespace artillery._ Thank you for your understanding!**

Basically, this crazy roundtype is meant to encourage **some form of cooperation between crew members to create the illusion of a "perfectly operating" station.** God help you.

But at least it's a rare roundtype.

**Comment by ShadowDeath6** [comment]
> rp gamemode
> with our playerbase

Good one

**Comment by ParagonPrime** [comment]
Looks as if the MediumRP option is going to win the vote on the forums, so... who knows :^)

and besides, stop nitpicking one detail

**Comment by ShadowDeath6** [comment]
Remove shitty rp job instead of making it an asshole job, its purpose is to debate about brig times and contest shitty judgements made by security, not whining about shit and trying to get a centcomm official with no real power spawned

**Comment by Alblaka** [comment]
> empower the Lawyer.

I would really like Lawyer to become a adminRPbus machine similar to Chaplain, but with a buerocratic and less religious theme. 
Still requires RPbussy admins to be online, but would certainly make the role more worthwile.

**Comment by Robertruler77** [comment]
> Implying that there aren't people who want to play a version of lawyer and it needs removed.

No but really, Please, lets make this a thing eventually. I wanna have power as a lawyer and not be a hanger oner in sec that inevtiably ends up in a compromising situation. (dead, Sec officer Promotion, Antag, etc.)

**Comment by Super3222** [comment]
Combine this with https://github.com/yogstation13/yogstation-old/issues/1175!!
